bindings/cpp/CMakeLists.txt

```diff
@@ -83,7 +80,9 @@ endif()
 execute_process(COMMAND cargo locate-project --workspace --message-format plain
```

Comment:
Additionally, I am considering whether we can first simply add a CI for Windows platform compilation and C++ unit test execution, so as to initially and easily ensure that compilation changes are reliable.